### PR TITLE
fix symlinking extensions into .k8slens/extensions folder

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: src/extensions/extension-discovery.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export class ExtensionDiscovery {
     // chokidar works better than fs.watch
     chokidar.watch(this.localFolderPath, {
       // Dont watch recursively into subdirectories
-      depth: 0,
+      depth: 1,
```
Comment (chenhunghan) [inline src/extensions/extension-discovery.ts:117]: Could you explain a little bit more (in the PR description) why this fix symlinking? There seems to be some discussion in Slack, but maybe some team member (like me) didn't fully follow.

Comment (panuhorsmalahti) [reply src/extensions/extension-discovery.ts:117]: Could add comment to the code explaining that depth: 1 is required for symlinking to work.

**Review by panuhorsmalahti — APPROVED** [review]

File: src/extensions/extension-discovery.ts (hunk 1 above)
Comment (panuhorsmalahti) [reply src/extensions/extension-discovery.ts:117]: @nevalla also noticed that depth: 1 is required in Windows

Comment (Nokel81) [reply src/extensions/extension-discovery.ts:117]: Should I add that to the comment as well?

Comment (panuhorsmalahti) [reply src/extensions/extension-discovery.ts:117]: Not necessarily I guess, just FYI.

**Pull request merged**